tests/integration/synapseclient/models/async/test_file_async.py

```diff
@@ -1225,7 +1225,7 @@ async def test_delete(
         # THEN I expect the file to be deleted
         with pytest.raises(SynapseHTTPError) as e:
             await file.get_async()
-        assert f"404 Client Error: \nEntity {file.id} is in trash can." in str(e.value)
+        assert f"404 Client Error: Entity {file.id} is in trash can." in str(e.value)
```

Comment:
Not exactly sure why the newline character is removed in the real exception being raised. The rest of the exception still has the expected new lines in it.

Reply:
I might have also slightly altered the format in https://github.com/Sage-Bionetworks/synapsePythonClient/blob/SYNPY-1476-remove-thread-executor-usage/synapseclient/core/exceptions.py#L189-L279 when I wrote it a few months ago, and now when we've swapped to HTTPX